paper-icon-button-light.html

```diff
@@ -45,6 +44,12 @@
         opacity: 0.6;
         color: currentColor;
       }
+      
+      img {
+        vertical-align: middle;
+        width: var(--iron-icon-width, 24px);
```

Comment:
Given that this is styling distributed content, we don't need to provide a variable since it can be styled from the outer tree.